addons/base_address_extended/models/res_partner.py

```diff
@@ -4,7 +4,7 @@
 from odoo import api, fields, models, tools
 
 
-class Partner(models.Model):
+class ResPartner(models.Model):
     _inherit = ['res.partner']
```

Comment:
This is done in the `(fix)` commit, can't the script detect that `_inherit = [...]` is the same as `_name = ...` when there is no `_name`?

It would certainly reduce the manual work needed in the post-script

Reply:
The script already does enough. It is not at all necessary to do this kind of detail too. Especially since I have already corrected them.

Reply:
> Especially since I have already corrected them.

What about partner code, forward ports, etc?